### PR TITLE
Add advanced menu items for expand collapse all

### LINKED ISSUE #9928: [Feature]: Possibility to open and close structure tree completely
### Feature description

In deeper nested structures it would be helpful to simply collapse/expand all nodes via the tree structures.

### PR DESCRIPTION
**Pull request by matthiashamacher**
## Changes in this pull request  
Resolves #9928

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/object/tree.js [hunk 1]
```diff
@@ -654,6 +654,27 @@ pimcore.object.tree = Class.create({
                 }
             }
 
+            // expand and collapse complete tree
+            if (record.data.type === "folder") {
```
Comment (brusch) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/tree.js]:
```suggestion
            if (!record.data.leaf) {
```
Also objects and variants can have children.

File: bundles/AdminBundle/Resources/public/js/pimcore/document/tree.js [hunk 2]
```diff
@@ -865,6 +865,27 @@ pimcore.document.tree = Class.create({
                 }
             }
 
+            // expand and collapse complete tree
+            if (record.data.type === "folder") {
```
Comment (brusch) [inline bundles/AdminBundle/Resources/public/js/pimcore/document/tree.js]:
```suggestion
            if (!record.data.leaf) {
```
Also pages can have children 😉

File: bundles/AdminBundle/Resources/public/js/pimcore/asset/tree.js [hunk 3]
```diff
@@ -770,6 +770,27 @@ pimcore.asset.tree = Class.create({
                 }
             }
 
+            // expand and collapse complete tree
+            if (record.data.type === "folder") {
```
Comment (brusch) [inline bundles/AdminBundle/Resources/public/js/pimcore/asset/tree.js]:
```suggestion
            if (!record.data.leaf) {
```

**Review by brusch — CHANGES_REQUESTED** [review]
Please see my inline comments on the code. 
Please also remove all translations except the one for `en`. Translations are managed here: https://poeditor.com/join/project/VWmZyvFVMH

Thanks!

**Comment by matthiashamacher** [comment]
I've implemented your requested changes.

**Comment by brusch** [comment]
Nice, thanks a lot! 👍

**Pull request merged**